client/src/Pages/Dashboard/DashboardView.js

```diff
@@ -1,72 +1,144 @@
 import React, { Component } from "react";
-import GoogleMap from "../../Components/GoogleMap/GoogleMap";
-import AntdTableComponent from "../../Components/AntdTable/AntdTableComponent";
+import { graphql } from "react-apollo";
+import { getDashboardAllDataQuery } from "../../queris/queris";
+
 import { Card } from 'semantic-ui-react';
-// import CarList from "../../Components/CarList"
-import CarNameCard from "../../Components/Card/CarNameCard";
-import LastFillupCard from "../../Components/Card/LastFillupCard";
-// import LastFillupTimeCard from "../../Components/Card/LastFillupTimeCard";
+import LoadingSpinner from '../../Components/LoadingSpinner';
+import LastFillUpCard from "../../Components/Card/LastFillUpCard";
 import FuelLeftCard from "../../Components/Card/FuelLeftCard";
-// import TraveldSinceCard from "../../Components/Card/TraveldSinceCard";
 import DiagnosticCard from "../../Components/Card/DiagnosticCard";
-// import DiagnosticDetailCard from "../../Components/Card/DiagnosticDetailCard";
 import BusinessRatioCard from "../../Components/Card/BusinessRatioCard";
-// import BusinessTotalCard from "../../Components/Card/BusinessTotalCard";
 import AverageSpeedCard from "../../Components/Card/AverageSpeedCard";
 import TravelDistanceTotalCard from "../../Components/Card/TravelDistanceTotalCard";
-// import TravelDistanceThisYearCard from "../../Components/Card/TravelDistanceThisYearCard";
 import TimeInCarCard from "../../Components/Card/TimeInCarCard";
 import EmissionsCard from "../../Components/Card/EmissionsCard";
 import FuelEconomyCard from "../../Components/Card/FuelEconomyCard";
-// import ParkingCard from "../../Components/Card/ParkingCard";
-// import TimeTraveldCard from "../../Components/Card/TimeTraveldCard";
-// import StartLocationCard from "../../Components/Card/StartLocationCard";
-// import EndLocationCard from "../../Components/Card/EndLocationCard";
-import SmartCard from "../../Components/Card/SmartCard";
 
+import GoogleMap from "../../Components/GoogleMap/GoogleMap";
+import AntdTableComponent from "../../Components/AntdTable/AntdTableComponent";
 
 
 class DashboardView extends Component {
+
+  constructor(props) {
+    super(props)
+  }
+
+  displayLastFillUpData() {
```

Comment:
this methods seem unnecessary. You can directly pass the value as a prop

Reply:
Right.. 
I thought I need a constructor() and super in order to pass the props inside render() method.

Reply:
Not sure what you mean. You can do this:
```
<AverageSpeedCard speed={this.props.data.car.averageSpeed} />
```

There is no value in these methods.

Reply:
Right! Cleaned up the unnecessary functions and simply just passing the props.